### DISCUSSION TITLE
ARC / SRS Implementation Resources

### DISCUSSION THREAD

**Discussion by developStorm** [simple-login/app#504] [Feature Requests]
Not sure I can make them, just put a post here to track the resources I've seen

**Comment by developStorm** [comment]
srslib (Python): https://srslib.readthedocs.io/en/latest/
Website: http://www.open-spf.org/SRS/

**Comment by developStorm** [comment]
ARC Website: https://arc-spec.org/
RFC 8617 (ARC): https://www.rfc-editor.org/info/rfc8617
ARC Recommended Usage (Inactive): https://datatracker.ietf.org/doc/draft-ietf-dmarc-arc-usage/ 
ARC Mailing List: http://lists.dmarc.org/mailman/listinfo/arc-discuss

**Comment by developStorm** [comment]
Real use case of ARC / SRS in alias service:
https://github.com/forwardemail/free-email-forwarding/issues/137
https://github.com/forwardemail/free-email-forwarding/blob/master/index.js#L1332

License: [Modified](https://github.com/forwardemail/free-email-forwarding/blob/master/LICENSE) Apache 2.0

**Comment by developStorm** [comment]
https://toolbox.googleapps.com/apps/messageheader/analyzeheader
https://github.com/mailhog/MailHog

**Comment by xyxzxyz** [comment]
I would love to see SimpleLogin implement ARC
